docs/getting-started/configuration.md

```diff
@@ -187,10 +189,77 @@ components: {
  },
 ```
 
+### Add sign-in resolver in the backend
+
+The Auth backend plugin is responsible for assigning a [Backstage User Identity](../auth/identity-resolver.md#backstage-user-identity) when a user signs in. The identity is mainly used in determining the ownership of Backstage Catalog entities for the user. This is achieved by writing a [Sign In Resolver](../auth/identity-resolver.md#sign-in-resolvers) which takes care of assigning the right identity to the user and even reject SignIn attempts from unrecognized email domains.
+
 > Since [v1.1.0](https://github.com/backstage/backstage/releases/tag/v1.1.0-next.3), you must provide an [explicit sign-in resolver](../auth/identity-resolver.md).
 
-That should be it. You can stop your Backstage App. When you start it again and
-go to your Backstage portal in your browser, you should have your login prompt!
+For now, let's create a simple Sign In Resolver which uses the first part of the user's email address to assign a Backstage User Identity.
```

Comment:
We actually just identified as part of the setup simplification work that we should have a very simple guest sign-in resolver. One that treats every single user that sign in as the same "guest" identity.

What do you think about adding that to its own section in `../auth/identity-resolver.md`, and then linking to that section from here? Kinda hate maintaining example codeblocks for the same thing in multiple places. We keep seeing again and again that they start to diverge and end up outdated.

Reply:
Here it is #11937 :grin:

Reply:
Amazing! @Rugvip When do you think https://github.com/backstage/backstage/pull/11937 is going to be released?

Let me update this PR with a note that you don't need to do anything with sign in resolvers as getting started as it comes with a guest resolver.

Reply:
Gonna be released tomorrow :grin: